addons/grenades/functions/fnc_flashbangExplosionEH.sqf

```diff
@@ -134,8 +134,10 @@ if (hasInterface && {!isNull ACE_player} && {alive ACE_player}) then {
         }, [], 17 * _strength] call CBA_fnc_waitAndExecute;
     };
 
-    if (_strength > 0.2) then {
-        ACE_player setVectorDir (_eyePos vectorDiff _grenadePosASL);
-    };
+    // Make player flinch
```

Comment:
```suggestion
    // Make player flinch
    if (_strength <= 0.2) exitWith {};
```

Reply:
Less for performance, more just to prevent doing extra `setDir`
I figure we need to think about how it will work with that 9-flash going off